.babelrc

```diff
@@ -1,5 +1,14 @@
 {
   "presets": [
     "env"
+  ],
+  "plugins": [
+    [
+      "transform-runtime",
+      {
+        "polyfill": false,
+        "regenerator": true
+      }
+    ]
   ]
-}
+}
```

Comment:
let's add an `empty line` (your IDE should have a plugin to automatically do this whenever a file is saved)